packages/cli/commands/logs.js

```diff
@@ -63,7 +63,7 @@ const endpointLog = async (accountId, options) => {
       getFunctionLogs(accountId, functionPath, { after });
     const fetchLatest = () => {
       try {
-        getLatestFunctionLog(accountId, functionPath);
+        return getLatestFunctionLog(accountId, functionPath);
```

Comment:
Seems reasonable, do we need to return in the catch as well?

Reply:
Doubled checked with @miketalley on this: I believe the intent is for polling to continue and `handleLogsError()` simply logs an error as a side effect.